src/datepicker/datepicker.spec.ts

```diff
@@ -23,13 +26,43 @@ function getDatepicker(element: HTMLElement): HTMLElement {
   return element.querySelector('ngb-datepicker') as HTMLElement;
 }
 
+function expectSameValues(datepicker: NgbDatepicker, config: NgbDatepickerConfig) {
+  expect(datepicker.dayTemplate).toBe(config.dayTemplate);
+  expect(datepicker.firstDayOfWeek).toBe(config.firstDayOfWeek);
+  expect(datepicker.markDisabled).toBe(config.markDisabled);
+  expect(datepicker.minDate).toBe(config.minDate);
+  expect(datepicker.maxDate).toBe(config.maxDate);
+  expect(datepicker.showNavigation).toBe(config.showNavigation);
+  expect(datepicker.showWeekdays).toBe(config.showWeekdays);
+  expect(datepicker.showWeekNumbers).toBe(config.showWeekNumbers);
+  expect(datepicker.startDate).toBe(config.startDate);
+}
+
+function customizeConfig(config: NgbDatepickerConfig) {
+  config.dayTemplate = {} as TemplateRef<DayTemplateContext>;
+  config.firstDayOfWeek = 2;
+  config.markDisabled = (date) => false;
+  config.minDate = {year: 2000, month: 0, day: 1};
+  config.maxDate = {year: 2030, month: 1, day: 31};
```

Comment:
Do we want to hard code this as a value or can we calculate it as added time from the `minDate`?  The reason I ask is because in other calendar systems, 01JAN is not the first of the year.

This _may_ mean that we need to have a base config object and have different calendars have different extensions of the config - which I'm ok with.  @pkozlowski-opensource @jnizet, thoughts?